tensorflow_io/oss/kernels/ossfs/oss_file_system.cc

```diff
@@ -12,17 +12,16 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-
 #include <pwd.h>
 #include <unistd.h>
 #include <cstdlib>
 #include <ctime>
 #include <fstream>
 #include <iostream>
 #include <vector>
+#include <cmath>
 
 #include "aos_string.h"
-#include "minIni.h"
```

Comment:
Now we don't require minIni, could delete third_party/minini.BUILD. Also modify WORKSPACE to remove the corresponding http_archive

Reply:
additionally, we remove minini dependency from tensorflow/oss/BUILD